CONTRIBUTION.md

```diff
@@ -0,0 +1,103 @@
+# Contribute to DB4S
+
+Thanks for your interest in our project.  
+This document is intended to serve as lighthouse for contributors, not an executor.
+
+## Contents
+- [How to ask about DB4S](#how-to-ask-about-db4s)
+- [How to contribute?](#how-to-contribute)
+    - [Report bug](#report-bug)
+    - [Suggest enhancement](#suggest-enhancement)
+    - [Translation](#translation)
+- [Styleguide](#styleguide)
+    - [CPP source code](#cpp-source-code)
+    - [Git commit message](#git-commit-message)
+
+## How to ask about DB4S?
+If you have any questions about our project, please check the following first.
+
+- Look up related topics on the [wiki](https://github.com/sqlitebrowser/sqlitebrowser/wiki)
+- Search for existing similar issues
+
+If your question is still not resolved, you can create an issue and ask a question.  
+A template from the GitHub issue will help you write a good bug report.
+
+## How to contribute?
+### Report bug/issue
+Please check the following before reporting a bug or issue.
+
+- Look up ['Known Issues'](https://github.com/sqlitebrowser/sqlitebrowser/issues/1688)
+- Search for existing similar issues
+- Try a [nightly build](https://nightlies.sqlitebrowser.org/latest)
+
+If a bug has not yet been reported or has not been resolved, please follow these steps to help members identify the bug.
+
+- Use a concise but descriptive title to identify the bug/issue
+    - Ex) `Crash DB4S when try to reset the settings`
+- Detailed steps to reproduce the bug
+    - Ex) `File -> Preferences -> General -> Restore Defaults`
+- What you actually expected to do and why
+    - Ex) `DB4S settings are initalized normally.`
+- System environment, DB4S version, loaded extensions
+    - Ex) `macOS Catalina Version 10.15.6(19G2021), DB4S Version 3.12.0, No Extensions`
+- Attach screenshots or screencasts related to the bug/issue
+    - You can add images can be pasted directly into the GitHub issue,  
+    but screencast can be hosted on streamable.com and the link pasted into the issue.
+- Crash log (If the crash log is known, although this is **NOT** required)
+    - If you have a long crash log, [Pastebin](https://pastebin.com/) can help (Make sure to set expiration to 'never')
+- Project file or database file in use (**NOT** required)
```

Comment:
Maybe we should add a reference to attach or paste the content of the Error Log in the SQL Log dock (and we maybe should rename that dock). Now it includes Qt errors as well as SQLite errors.

Reply:
I wrote it like below, What do you think?
>SQL log (If the SQL log is known, although this is **NOT** required)

Reply:
What do you mean by "If the SQL log is known"?

Reply:
I mean the case where there is an SQL log.

Reply:
Maybe "If the log content is relevant"? There would be cases when that is not the case.

Reply:
That would be perfect, I modified it like below!
>SQL log (If the log content is relevant)